### PR TITLE
Add OPENUSD and USDOPEN as price identifiers

### PR DESCRIPTION
**Pull request by OpenDAO-Logan**

### CODE REVIEW

File: UMIPs/umip-draft_OPEN_price_identifier_v2.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+## Headers
+
+| **UMIP #** |
```
Comment (smb2796) [inline UMIPs/umip-draft_OPEN_price_identifier_v2.md:3]: This header formatting is messed up. Mind taking another look?

File: UMIPs/umip-draft_OPEN_price_identifier_v2.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+- **Markets & Pairs:**  
+  - *Uniswap*: OPEN/ETH 
+  - *Uniswap*: OPEN/USDT
+- **Example data providers:** Uniswap - Direct
+- **Cost to use:** Free
+- **Real-time data update frequency: Frequency** - On every Ethereum block (i.e. every ~15 seconds)
+- **Historical data update frequency: Frequency** - hourly
+
+# Price Feed Implementation
+
```
Comment (smb2796) [inline UMIPs/umip-draft_OPEN_price_identifier_v2.md:48]: Can you wrap this block of code in triple tick marks, like so:

```
pfConfig
```

**Comment by smb2796** [comment]
Couple of small formatting things, but this generally looks good to me!

File: UMIPs/umip-draft_OPEN_price_identifier_v2.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+## Headers
```
Comment (mhairimca) [inline UMIPs/umip-draft_OPEN_price_identifier_v2.md:1]: looks good

File: UMIPs/umip-draft_OPEN_price_identifier_v2.md (hunk 1 above)
Comment (OpenDAO-Logan) [reply UMIPs/umip-draft_OPEN_price_identifier_v2.md:3]: will do!

Comment (OpenDAO-Logan) [reply UMIPs/umip-draft_OPEN_price_identifier_v2.md:3]: Got it. I needed to put two spaces after each section in the heading in order to give them a line break... my bad.

**Review by smb2796 — APPROVED** [review]
The header format is still a little wonky, but I can fix on my side. Other than that, this price id looks good. Merging in and moving to `Last Call`.

**Pull request merged**